test/test_core.rb

```diff
@@ -653,18 +653,23 @@ def test_namespace
       assert_equal "<sean:blah>Some text</sean:blah>", out
     end
 
-
     def test_add_namespace
       e = Element.new 'a'
+      assert_equal "", e.namespace
+      assert_equal nil, e.namespace('foo')
```

Comment:
Could you avoid omitting parentheses for assertions to avoid potential warning?
See also: https://bugs.ruby-lang.org/issues/20922

```suggestion
      assert_equal("", e.namespace)
      assert_equal(nil, e.namespace('foo'))
```

Reply:
OK.
I see.